config/routes.rb

```diff
@@ -20,4 +22,7 @@
   resources :carts, only: :index do
     post :remove_cart_items, on: :collection
   end
+
+  resources :account_activations, only: [:edit]
```

Comment:
You don't need [ ] if there's only one action, and for the one below, you can use the `%i` like you did above.